src/Api/Models/Response/ConfigResponseModel.cs

```diff
@@ -47,6 +48,7 @@ public class ServerConfigResponseModel
 public class EnvironmentConfigResponseModel
 {
     public string Vault { get; set; }
+    public string CloudVault { get; set; }
```

Comment:
💭 This name gives me pause and I think your team could chat on alternatives. Other names like "admin" or "management" come to mind and we should certainly ensure this is documented well for general usage.

Reply:
I have made a change to the code by renaming it to `CloudWebVault` to ensure consistency between the server and clients. 
The purpose of this URL is to reference the general cloud vault (vault.bitwarden.com/eu), while we already have the `Vault` environment variable. By using the name `CloudWebVault`, we can distinguish between these two variables more clearly.